src/server/api/controllers/days-to-end-batch.controllers.js

```diff
@@ -0,0 +1,31 @@
+const knex = require('../../config/db');
+
+const Error = require('../lib/utils/http-error');
+
+const moment = require('moment');
+
+const getDayLeftToEndBatch = async (batchId) => {
+  try {
+    const batch = await knex('batches')
```

Comment:
To avoid having `batch[0]` everywhere, we can have this, which means `const batch = result[0]`, or taking the first element of the result.
```suggestion
    const [batch] = await knex('batches')
```